pipelines/datasets/br_fnde_fundeb/README.md

```
@@ -0,0 +1,371 @@
+
+| Produto | Conteúdo | Arquivo |
+|---|---|---|
+| 53 | 2021 a 2024 | `Indicadores_SIOPE_ate_2024.txt.gz` |
+| 54 | exercício corrente | `Indicadores_SIOPE.txt.gz` |
+
+Páginas: [produto 53](https://www.fnde.gov.br/plataforma-antonieta-de-barros/dados/produtos-de-dados/visualizar/53)
+e [produto 54](https://www.fnde.gov.br/plataforma-antonieta-de-barros/dados/produtos-de-dados/visualizar/54).
+
+O portal é uma SPA — a página não carrega link de download. O arquivo sai da API:
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)